scripts/lib/install

```diff
@@ -99,17 +99,21 @@ export LANG="en_US.UTF-8"
 export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
-apt-get install -y lsb-release
-os_info="$(lsb_release --short --id --release --codename)"
-{ read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+if [ -f /etc/os-release ]; then
+    os_info="$(. /etc/os-release; printf '%s\n' "$ID" "$VERSION_ID" "$VERSION_CODENAME")"
+    { read -r os_id; read -r os_version_id; read -r os_version_codename; } <<< "$os_info"
+else
+    export os_id="unsupported"
+    export os_version_id="unsupported"
```

Comment:
Nit: no need to `export` these.

Reply:
Fixed.

Reply:
What I meant was `os_id="unsupported"` instead of `export os_id="unsupported"`.

Reply:
(Though I guess I’d also be fine with deleting the `if` and having `. /etc/os-release` just fail with the already clear-enough error message `/etc/os-release: No such file or directory` if you’re on a sufficiently weird system.)

Reply:
There is a check of `if [ -f /etc/os-release ]; then` before reading the content of `/etc/os-release`, so what will happen is that the message "Unsupported OS release ..." will still appear.